content/applications/productivity/iot/config/windows_iot.rst

```diff
@@ -0,0 +1,180 @@
+installing the Windows virtual IoT software for the first time.
+
+Download and initial installation
+---------------------------------
+
+To begin the installation, navigate to the Odoo 16 or higher installation package for Enterprise or
+Community Windows at `Odoo's Download Page <https://odoo.com/download>`_. Next, Install and setup
+the Odoo `.exe` file. After the instructions screen, click :guilabel:`Next` to start the
+installation and agree to the :abbr:`TOS (Terms of Service)`.
+
+During the next step of the installation, select :guilabel:`Odoo IoT` from the :guilabel:`Select the
+type of install` dropdown.
+
+.. example::
+   For reference, the following should be installed:
+
+   - **Odoo server**
+   - **Odoo IoT**
+   - **Nginx WebServer**
+   - **Ghostscript interpreter**
+
+Ensure there is enough space on the computer for the installation and click :guilabel:`Next`.
+
+Setting the destination and completing the installation
+-------------------------------------------------------
+
+To complete the installation, select the :guilabel:`Destination Folder`. Choosing ``C:\odoo`` as the
+install location will allow for the Nginx server to start. Click :guilabel:`Install`. If the folder
+doesn't exist, then create it. Otherwise the installation files will be spread throughout the hard
+drive. Allow the installation to occur. This may take a few minutes. Click :guilabel:`Next` to
+continue.
+
+.. warning::
+   Odoo's Windows virtual IoT software shouldn't be installed inside any of the Window's User's
+   directories. Doing so won't allow for Nginx to initialize.
+
+Ensure that the :guilabel:`Start Odoo` box is checked and click :guilabel:`Finish`. After
+installation, the Odoo server will run and automatically open `http://localhost:8069` on your web
+browser. The webpage should display the :abbr:`IoT (Internet of Things)` *box home page*
+
+.. seealso::
+   A restart of the Windows IoT program may be necessary should the web browser not display
+   anything. :ref:`restart_windows_iot`
+
+Connecting devices
+------------------
+
+Next, connect the :abbr:`IoT (Internet of Things)` devices to the Windows computer. Windows should
+automatically detect the device because the driver is pre-installed on the computer. If not, search
+for and install the Windows driver for the device.
+
+Following connecting devices to the computer, refresh the :abbr:`IoT (Internet of Things)` *box
+home page* and verify the device is seen. If not, reload the handlers through the :abbr:`IoT
+(Internet of Things)` *box home page*.
+
+Finally, connect Windows :abbr:`IoT (Internet of Things)` to a database using existing instructions
+(manually using the Token).
+
+.. seealso::
+   :doc:`connect`
+
+Now the installation is complete, the devices connected to :abbr:`IoT (Internet of Things)` can be
+used to complete processes/actions.
+
+Troubleshooting
+===============
+
+.. _restart_windows_iot:
```

Comment:
please add the app name to the custom anchor (don't forget to update the corresponding :ref: tag)
```suggestion
.. _iot/restart_windows_iot:
```